### ISSUE TITLE
Adding manpages breaks the RPM build

### ISSUE THREAD

**Issue by davidozog** [Sandia-OpenSHMEM/SOS#425]
`rpmbuild` is broken by PR #413:
```
RPM build errors:
    Installed (but unpackaged) file(s) found:
   /usr/share/man/man1/oshc++.1.gz
   /usr/share/man/man1/oshcc.1.gz
   /usr/share/man/man1/oshfort.1.gz
   /usr/share/man/man1/oshrun.1.gz
   /usr/share/man/man3/_my_pe.3.gz
   /usr/share/man/man3/_num_pes.3.gz
```
Attempted command:
```
rpmbuild -ta sandia-openshmem-1.3.3rc1.tar.gz --define "configargs --prefix=/home/install/SOS-manpages --enable-pmi-simple"
```
The problem is within the spec file: the manpage files are not included in the %files section.  It's probably not appropriate to add them manually, because the %makeinstall directive doesn't appear to be doing the right thing.  We may need to include a manual make install, as in:
```
make DESTDIR=${RMP_BUILD_ROOT} install
```

**Comment by jdinan** [comment]
@matcabral Any suggestions?

**Comment by davidozog** [comment]
Closed by #426
